docs/user-guides/configurations.md

```diff
@@ -66,21 +66,23 @@ Self-optimizing configurations are applicable to both Iceberg Format and Mixed s
+| history.expire.min-snapshots-to-keep        | 1               | Minimum number of snapshots retained for table expiration                                                                                                                                                                                                             |
+| clean-orphan-file.enabled                   | false           | Enables periodically clean orphan files                                                                                                                                                                                                                               |
+| clean-orphan-file.min-existing-time-minutes | 2880(2 days)    | Cleaning orphan files keeps the files modified within a specified time in minutes                                                                                                                                                                                     |
+| clean-dangling-delete-files.enabled         | true            | Whether to enable cleaning of dangling delete files                                                                                                                                                                                                                   |
+| data-expire.enabled                         | false           | Whether to enable data expiration                                                                                                                                                                                                                                     |
+| data-expire.level                           | partition       | Level of data expiration. Including partition and file                                                                                                                                                                                                                |
+| data-expire.field                           | NULL            | Field used to determine data expiration, supporting timestamp/timestampz/long type and string type field in date format                                                                                                                                               |
+| data-expire.datetime-string-pattern         | yyyy-MM-dd      | Pattern used for matching string datetime                                                                                                                                                                                                                             |
+| data-expire.datetime-number-format          | TIMESTAMP_MS    | Timestamp unit for long field. Including TIMESTAMP_MS and TIMESTAMP_S                                                                                                                                                                                                 |
```

Comment:
I think it is compatible with iceberg itself  [Table behavior properties],WDY?(https://iceberg.apache.org/docs/1.6.1/configuration/#table-behavior-properties)